graphql/id.go

```diff
@@ -20,6 +20,8 @@ func UnmarshalID(v interface{}) (string, error) {
 		return string(v), nil
 	case int:
 		return strconv.Itoa(v), nil
+	case int64:
+		return strconv.Itoa(int(v)), nil
```

Comment:
how is https://golang.org/pkg/strconv/#FormatInt ?

Reply:
@vvakame I opted to use strconv.Itoa to be consistent with the UnmarshalIntID function. It looks like Itoa is just an alias to FormatInt(int64(x), 64):
https://golang.org/pkg/strconv/#Itoa

If you want I can update both to use strconv.FormatInt explicitly though.

Reply:
Done